metering/metering-upgrading-metering.adoc

```diff
@@ -0,0 +1,143 @@
+
+*  The cluster is updated to 4.5.
+*  The xref:../metering/metering-installing-metering.adoc#metering-install-operator_installing-metering[Metering Operator] is installed from OperatorHub.
++
+[NOTE]
+====
+You must upgrade the Metering Operator to {product-version} manually. Metering does not upgrade automatically if you selected the "Automatic" *Approval Strategy* in a previous installation.
+====
+*  The xref:../metering/configuring_metering/metering-about-configuring.adoc#metering-about-configuring[MeteringConfig] is configured.
+*  The xref:../metering/metering-installing-metering.adoc#metering-install-metering-stack_installing-metering[metering stack] is installed.
+*  Ensure that metering status is healthy by checking that all Pods are ready.
+
+[IMPORTANT]
+====
+Potential data loss can occur if you modify your metering storage configuration after installing or upgrading metering.
+====
+
+// include::modules/metering-upgrade-operator.adoc[leveloffset=+1]
+// include::modules/metering-upgrade-verify.adoc[leveloffset=+1]
+
+.Procedure
+
+.  Click *Operators* -> *Installed Operators* from the web console.
+
+.  Select the `openshift-metering` Project.
+
+.  Click *Metering Operator*.
+
+.  Click *Subscription* -> *Channel*.
+
+.  In the *Change Subscription Update Channel* window, select *{product-version}* and click *Save*.
++
+[NOTE]
+====
+Wait several seconds to allow the subscription to update before proceeding to the next step.
+====
+.  Click *Operators* -> *Installed Operators*.
++
+The Metering Operator is shown as 4.5. For example:
++
+----
+Metering
+4.5.0-202005202015 provided by Red Hat, Inc
+----
+
+.Verification
+You can verify the metering upgrade by performing any of the following:
+
+*  Check the Metering Operator ClusterServiceVersion (CSV) for the new metering version. This can be done through either the web console or CLI.
++
+--
+.Procedure (UI)
+  .  Navigate to *Operators* -> *Installed Operators* in the metering namespace.
+  .  Click *Metering Operator*.
+  .  Click *Subscription* for *Subscription Details*.
+  .  Check the *Installed Version* for the upgraded metering version. The *Starting Version* shows the metering version prior to upgrading.
+
+.Procedure (CLI)
+*  Check the Metering Operator CSV:
++
+----
+$ oc get csv
+----
++
+In the following example, the 4.5 Metering Operator upgrade is successful and replaces the 4.4 metering installation:
++
+----
+NAME                                        DISPLAY                  VERSION              REPLACES                               PHASE
+elasticsearch-operator.4.5.0-202006161654   Elasticsearch Operator   4.5.0-202006161654                                          Succeeded
+metering-operator.4.5.0-202006161654        Metering                 4.5.0-202006161654   metering-operator.4.4.0-202005252114   Succeeded
+----
+--
+
+*  Check that all required Pods in the metering namespace are created. This can be done through either the web console or CLI.
```

Comment:
Do we need to substitute this with `... openshift-metering namespace`?

Reply:
I actually went back and forth on that. I changed it ... in the `openshift-metering` namespace ...